msg/FailsafeFlags.msg

```diff
@@ -25,9 +26,9 @@ bool attitude_invalid                 # Attitude invalid
 bool local_altitude_invalid           # Local altitude invalid
 bool local_position_invalid           # Local position estimate invalid
 bool local_position_invalid_relaxed   # Local position with reduced accuracy requirements invalid (e.g. flying with optical flow)
-bool local_velocity_invalid           # Local velocity estimate invalid
+bool local_velocity_invalid           # Local velocity estimate invalid Additional
```

Comment:
Additional?

Reply:
🤣 Good catch. That was only meant as a note for me because I was going through which of the mode requirement masks also has a corresponding `_invalid` flag, the others are "Additional". Slipped into my commit, let me correct that 🤦

Reply:
I removed these notes in an interactive rebase on `main`.